### PR TITLE
replaced axios by native fetch

### PR DESCRIPTION
**Pull request by vovk1805**

### CODE REVIEW

**Comment by stursby** [comment]
What are the advantages considering `axios` has greater browser support and is already working??

File: src/App.vue [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <template v-if="repos">
       <h1 v-if="!tie">{{ vueHasPassedReact ? 'YES' : 'NO' }}</h1>
       <h1 v-else>TIE!</h1>
+      <p class="left" v-if="!vueHasPassedReact">left: {{ left }}</p>
```
Comment (stursby) [inline src/App.vue:8]: I don't think think this is needed? It already says how many stars away.

File: src/App.vue [hunk 2]
```diff
@@ -84,6 +84,9 @@ export default {
     },
     tie() {
       return this.vueStars === this.reactStars
+    },
+    left() {
```
Comment (stursby) [inline src/App.vue:88]: Same as above... don't think we need this?

File: src/App.vue (hunk 1 above)
Comment (vovk1805) [reply src/App.vue]: Really IE? replacing axios by native fetch will give us one more dependence less

**Comment by vovk1805** [comment]
Really IE? replacing axios by native fetch will give us one more dependence less

File: src/App.vue (hunk 1 above)
Comment (vovk1805) [reply src/App.vue:8]: I think it would be nice to see how many stars still remain "for pass"

File: src/App.vue (hunk 2 above)
Comment (vovk1805) [reply src/App.vue:88]: same here)

**Comment by vovk1805** [comment]
oh... sorry, didn't take attention on function folder. Could you please close PR?

**Comment by stursby** [comment]
Well, the functions is mostly independent (running on [Webtask](https://webtask.io/)) so could drop axios potentially. I just like axios and it works great and it pretty small, so doesn't bother me that it's another dep.

**Comment by vovk1805** [comment]
yep, I like the axios too. But if I can do something by native possibility of JS I doing it. For example ```document.querySelector``` instead ```$('some selector')```

**Pull request closed without merging**